tests/interpolation/test_custom_resolvers.py

```diff
@@ -8,22 +8,57 @@
 from tests.interpolation import dereference_node
 
 
-def test_register_resolver_twice_error(restore_resolvers: Any) -> None:
+def test_register_resolver_twice_error_new_callable(restore_resolvers: Any) -> None:
     def foo(_: Any) -> int:
         return 10
 
     OmegaConf.register_new_resolver("foo", foo)
-    with raises(AssertionError):
+    with raises(ValueError, match=re.escape("resolver 'foo' is already registered")):
         OmegaConf.register_new_resolver("foo", lambda _: 10)
```

Comment:
Since we no longer care if it's the same callable or not, I think we can just have a single test like:
```
test_register_resolver_twice_error
```

Reply:
Also done in e5a490b